packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/storage_columns.dart

```diff
@@ -126,13 +126,17 @@ class StorageSystemColumn extends StorageColumn {
             return Text(lang.diskHeadersSystem);
           },
           diskBuilder: (context, disk) {
-            return Text(disk.ptable ?? '');
```

Comment:
ptable was previously displayed just to have something there to avoid an empty column